pkgs/by-name/td/tdf/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  cairo,
+  glib,
+  poppler,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "tdf";
+  version = "0-unstable-2024-05-29";
+
+  src = fetchFromGitHub {
+    owner = "itsjunetime";
+    repo = "tdf";
+    fetchSubmodules = true;
+    rev = "51b49b01115d9e25c597c3c29768bf1069a6c6b2";
+    hash = "sha256-CWnB1e/EWHqSqiOpK2SjlbAry5GHwPlZNP9RQkLA4zQ=";
```

```suggestion
    rev = "1e89a93879fbb7a5b78f43fbb61d0c99ce184e4f";
    hash = "sha256-Zl5XAaEGTPbOjlFltCQqhPyEExk7Xs+NY4uNtagCgCw=";
```
Update past the license being added then LGTM

---

Done!